README.md

```
@@ -184,36 +154,36 @@ Special thanks to our amazing mentors who are guiding this project! 🙌
-                    <br />
-                    <sub><b>Vishnu Prasad Korada</b></sub>
-                </a>
-            </td>
             <td align="center">
                 <a href="https://github.com/sajalbatra">
                     <img src="https://avatars.githubusercontent.com/u/125984550?v=4" width="100;" alt="sajalbatra"/>
                     <br />
                     <sub><b>Sajal Batra</b></sub>
                 </a>
             </td>
+		</tr>
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)